digest/digest.go

```diff
@@ -49,7 +49,7 @@ func NewDigestFromHex(alg, hex string) Digest {
 }
 
 // DigestRegexp matches valid digest types.
-var DigestRegexp = regexp.MustCompile(`[a-zA-Z0-9-_+.]+:[a-zA-Z0-9-_+.=]+`)
+var DigestRegexp = regexp.MustCompile(`[a-zA-Z0-9-_+.]+:[a-fA-F0-9]+`)
```

Comment:
Josh and I discussed this, since it is not supported now the regexp should match what is supported. Allowing it now does not create any forward compatibility, it will just need to be updated to support base64 then.